### ISSUE TITLE
Keyboard LEDs don't work on Bonobo bonx6

### ISSUE THREAD

**Issue by dleary** [pop-os/system76-dkms#9]
**Distribution (run `cat /etc/os-release`):**
```
NAME="Pop!_OS"
VERSION="18.04 LTS"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 18.04 LTS"
VERSION_ID="18.04"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic

``` 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
```
system76-dkms:
  Installed: 0.0.3~1532711988~18.04~c523cdb
  Candidate: 0.0.3~1532711988~18.04~c523cdb
  Version table:
 *** 0.0.3~1532711988~18.04~c523cdb 1001
       1001 http://ppa.launchpad.net/system76/pop/ubuntu bionic/main amd64 Packages
        100 /var/lib/dpkg/status

```
**Issue/Bug Description:**
Bonobo model bonx6 keyboard LEDs get turned off at boot time and don't work thereafter.

**Steps to reproduce (if you know):**
After power on, up to the time where I'm prompted for the disk decryption key all the kb led functions work fine (Fn + numpad / or * or - or +).
As bootup proceeds however, the kb leds get turned off in conjunction with display of the following log messages:

```
Aug 31 09:36:46 dll76b kernel: [    9.822560] system76: loading out-of-tree module taints kernel.
Aug 31 09:36:46 dll76b kernel: [    9.822892] system76: module verification failed: signature and/or required key missing - tainting kernel
Aug 31 09:36:46 dll76b kernel: [    9.825321] system76: kb_led_resume
Aug 31 09:36:46 dll76b kernel: [    9.825671] system76: kb_led_disable
Aug 31 09:36:46 dll76b kernel: [    9.831598] system76: kb_led_color_set 0 FFFFFF
Aug 31 09:36:46 dll76b kernel: [    9.839631] system76: kb_led_color_set 1 FFFFFF
Aug 31 09:36:46 dll76b kernel: [    9.847387] system76: kb_led_color_set 2 FFFFFF
Aug 31 09:36:46 dll76b kernel: [    9.855628] system76: kb_led_color_set 3 FFFFFF
Aug 31 09:36:46 dll76b kernel: [    9.862317] system76: kb_led_set 0
Aug 31 09:36:46 dll76b kernel: [    9.867965] system76: kb_led_enable

```
Once fully booted and logged in, the kb led fn keys * - and + elicit a pop up window with an icon identifying the intended function but have no effect on the actual kb leds.  The following messages are simultaneously logged to /var/log/kern.log:

fn * off then on:
```
Aug 31 10:11:12 dll76b kernel: [ 2085.123565] system76: WMI event code (9e)
Aug 31 10:11:12 dll76b kernel: [ 2085.123568] system76: Unknown WMI event code (9e)
Aug 31 10:11:15 dll76b kernel: [ 2088.014907] system76: WMI event code (9f)
Aug 31 10:11:15 dll76b kernel: [ 2088.014910] system76: kb_wmi_brightness 0
Aug 31 10:11:15 dll76b kernel: [ 2088.014912] system76: kb_led_set 0

```
fn - - then fn + +:
```
Aug 31 10:12:07 dll76b kernel: [ 2140.441688] system76: WMI event code (81)
Aug 31 10:12:07 dll76b kernel: [ 2140.441692] system76: kb_wmi_brightness 192
Aug 31 10:12:07 dll76b kernel: [ 2140.441693] system76: kb_led_set 192
Aug 31 10:12:08 dll76b kernel: [ 2140.921418] system76: WMI event code (81)
Aug 31 10:12:08 dll76b kernel: [ 2140.921422] system76: kb_wmi_brightness 144
Aug 31 10:12:08 dll76b kernel: [ 2140.921423] system76: kb_led_set 144
Aug 31 10:12:09 dll76b kernel: [ 2142.135682] system76: WMI event code (82)
Aug 31 10:12:09 dll76b kernel: [ 2142.135686] system76: kb_wmi_brightness 192
Aug 31 10:12:09 dll76b kernel: [ 2142.135687] system76: kb_led_set 192
Aug 31 10:12:09 dll76b kernel: [ 2142.458955] system76: WMI event code (82)
Aug 31 10:12:09 dll76b kernel: [ 2142.458958] system76: kb_wmi_brightness 255
Aug 31 10:12:09 dll76b kernel: [ 2142.458960] system76: kb_led_set 255

```
The fn  / function doesn't do anything (no log message and no popup window).

**Expected behavior:**
Expect the kb led fn keys to work properly after pop_os has booted.

**Other Notes:**

**Comment by jackpot51** [comment]
I believe I have a fix for this in #10

**Comment by dleary** [comment]
Confirmed fixed as of
Version: 1.0.0~1536351270~18.04~93caafa
Thanks much.
